graphics/nine_patch.cpp

```diff
@@ -208,11 +239,18 @@ NinePatchBitmap::NinePatchBitmap(Graphics::TransparentSurface *bmp, bool owns_bi
 	}
 }
 
+void NinePatchBitmap::modifyTitleWidth(int titleWidth) {
+	if (_titlePos == 0) return;
+	_titleWidth = titleWidth;
+	_h.calcOffsets(_cached_dw, _titleIndex, _titleWidth);
+}
+
 void NinePatchBitmap::blit(Graphics::Surface &target, int dx, int dy, int dw, int dh, byte *palette, int numColors, MacWindowManager *wm, uint32 transColor) {
 	/* don't draw bitmaps that are smaller than the fixed area */
 	if (dw < _h._fix || dh < _v._fix)
 		return;
 
+	if (dw < _h._fix + _titleWidth) dw = _h._fix + _titleWidth;
```

Comment:
The same thing, try to avoid statements like this, they are not easy to read. Split it into two lines.